### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by Thialyson** [Sach97/scrapy.dart#8]
Say you need to add lib path_provider as a dependency.

**Comment by Sach97** [comment]
Yes it is an external dependency thus you have to add it to your pubspec.yaml yourself. Do you want me to explain this in the docs?

**Comment by micer** [comment]
Yep, would be great to add `path_provider` and `html` as dependencies on pub.dev and here to README. As a newbie in Flutter it took me a while to figure this out. Thanks!
